app/views/announcements/index.html.slim

```diff
@@ -1,5 +1,5 @@
 div.page-header
   h1
     = t('.header')
-= render partial: 'admin/announcements/announcement', collection: @announcements
+= render partial: 'system/admin/announcements/announcement', collection: @announcements
```

Comment:
I think currently global announcement is referencing the instance partial, you want the reverse way ?